topics/persistence.md

```diff
@@ -121,6 +129,14 @@ you'll end up with a single key in your dataset containing the final
 value, but 100 entries in your AOF. 99 of those entries are not needed
 to rebuild the current state.
 
+The rewrite is completely safe as while Redis continues appending to the old file, a completely new one is produced with the minimal set of operations needed to create the current data set, and once this second file is ready Redis switches the two and starts appending to the new one.
```

Comment:
```suggestion
The rewrite is completely safe. While Redis continues appending to the old file, a completely new one is produced with the minimal set of operations needed to create the current data set, and once this second file is ready Redis switches the two and starts appending to the new one.
```

Reply:
the line is quite long. it'l be easier for review, and future updates if we split it after some commas or alike, so adding a line will not create such a big diff.

Reply:
This is the previous document, do we need to divide them into shorter lines now?

Reply:
sorry, it looked new since it was moved.
i guess that since we moved it, we can take the opportunity and break it to shorter lines.
it's not a hard rule.. just a guideline for new parts we add.

Reply:
OK